main.go

```diff
@@ -56,15 +56,15 @@ func main() {
 			os.Exit(1)
 		}
 
-		certs, err := getCerts(*dumpFile, format)
+		certs, aliases, err := getCerts(*dumpFile, format)
```

Comment:
Don't make this separate return values, use a struct and group them together.

For example:
```
type CertWithAlias struct {
   alias string
   cert *x509.Certificate
}```